tests/test_node_ci_modernizer.py

```diff
@@ -0,0 +1,59 @@
+"""
+Tests for Github Actions Modernizer Script
+"""
+import os
+import shutil
+import uuid
+from unittest import TestCase
+
+from edx_repo_tools.codemods.node16 import GithubCINodeModernizer
+from edx_repo_tools.utils import YamlLoader
+
+
+class TestGithubActionsModernizer(TestCase):
+
+    def setUp(self):
```

Comment:
instead of manually doing a setup to create and destroy files we can use `tmpdir` pytest fixture to setup files temporarily that we don't have to remove later, as done in https://github.com/openedx/repo-tools/blob/master/tests/test_modernize_openedx_yaml.py#L9